reference-data-proxy/api-tests/v1/industry-sectors.api-test.ts

```diff
@@ -1,15 +1,13 @@
-const app = require('../../src/createApp');
-
+import { app } from '../../src/createApp';
 const { get } = require('../api')(app);
 
 describe('/industry-sectors', () => {
   describe('GET /industry-sectors', () => {
     it('returns a list of industry-sectors', async () => {
       const { status, body } = await get('/industry-sectors');
-
       expect(status).toEqual(200);
       expect(body.industrySectors.length).toBeGreaterThan(1);
-      body.industrySectors.forEach((industrySector) => {
+      body.industrySectors.forEach((industrySector: any) => {
```

Comment:
Would be nice to create custom type definition for industrySector rather than using any

Reply:
yup, I agree.
However we can change it further down the line :)